lobby/src/main/java/org/triplea/lobby/server/db/UserController.java

```diff
@@ -68,33 +59,18 @@ public void updateUser(final DBUser user, final HashedPassword hashedPassword) {
 
     try (Connection con = newDatabaseConnection();
         PreparedStatement ps = con.prepareStatement(
-            String.format("update ta_users set %s=?, email=? where username=?", getPasswordColumn(hashedPassword)))) {
+            "update ta_users set bcrypt_password=?, email=? where username=?")) {
       ps.setString(1, hashedPassword.value);
       ps.setString(2, user.getEmail());
       ps.setString(3, user.getName());
       ps.execute();
-      if (!hashedPassword.isBcrypted()) {
-        try (PreparedStatement ps2 =
-            con.prepareStatement("update ta_users set bcrypt_password=null where username=?")) {
-          ps2.setString(1, user.getName());
-          ps2.execute();
-        }
-      }
       con.commit();
     } catch (final SQLException e) {
-      throw newDatabaseException(String.format("Error updating name: %s, email: %s, (masked) pwd: %s",
-          user.getName(), user.getEmail(), hashedPassword.mask()), e);
+      throw newDatabaseException(String.format("Error updating name: %s, email: %s",
+          user.getName(), user.getEmail()), e);
```

Comment:
A few reasons actually:
- code attached to methods to support only logging is a bit of a code smell
- people reading the logs will not be confused by not seeing a lack of password. If we think the information *would* be very useful to know about errors, then we can handle it at the time of logging. Given we are dealing with a password, not even operating on it keeps the local scope smaller and easier to audit. It's already very questionable that we log email and full username (it can be recommend not logging full username for the cases when a user had the wrong fields focused and types their password as their username).